services/aws/index.js

```diff
@@ -0,0 +1,43 @@
+const AWS = require('aws-sdk');
+require('dotenv').config();
+
+const awsConfig = {
+  region: 'ap-south-1',
+  endpoint: process.env.AWS_DYNAMO_DB_ENDPOINT_URL,
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+};
+AWS.config.update(awsConfig);
+
+const docClient = new AWS.DynamoDB.DocumentClient();
+
+async function dailyCheckinsFeedback(params) {
+  const input = {
```

Comment:
Why are we using an extra variable, move this to payload

Reply:
done